data/002/0151/en_US.json

```diff
@@ -2,7 +2,7 @@
 	"002": {
 		"0151": {
 			"name": "Custom Pretendo Error",
-			"message": "",
+			"message": "The online service is unavailable.\nPlease try again later.\n\nFor help, visit support.nintendo.com.",
 			"short_description": "Error when registering new account",
 			"long_description": "Unknown cause.",
```

Comment:
Long description needed

Reply:
No one actually got this error before, so I have no idea what should be the long one. In the meantime we could just copy the short description until we know more how Pretendo wants to use this custom error code